pynicotine/config.py

```diff
@@ -180,7 +180,8 @@ def load_config(self):
                 "ipignorelist": {},
                 "ipblocklist": {},
                 "autojoin": ["nicotine"],
-                "autoaway": 15,
```

Comment:
Changing the data type of an existing config setting might lead to runtime errors if the user ever needs to revert back to an older version. Please test for backwards compatibility.

Perhaps ideally the setting could be converted to a tuple `(15, True)`, but I don't think doing this works when rolling back to older versions.

It would be easier for maintenance to keep the existing setting's name and create a new one for the CheckButton.

Reply:
Runtime errors which are silent and don't throw a critical error are the worst, because it can lead to mysterious behaviour.